### PR TITLE
argocd, argocd-app: add page

### PR DESCRIPTION
**Pull request by bataliero**
- [x] The page(s) are in the correct platform directories: `common` etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** v2.6.1

### CODE REVIEW

**Review by einverne — APPROVED** [review]
LGTM. Thank you for the new pages.

File: pages/common/argocd-app.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+> Command-line interface to manage applications by Argo CD.
+> More information: <https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_app/>.
+
+- List applications:
+
+`argocd app list --output {{json|yaml|wide}} `
+
+- Get application details:
+
+`argocd app get {{app-name}} --output {{json|yaml|wide}}`
```
Comment (blueskyson) [inline pages/common/argocd-app.md:12]: We use snake case.
```suggestion
`argocd app get {{app_name}} --output {{json|yaml|wide}}`
```

File: pages/common/argocd-app.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Deploy application internally (to the same cluster that Argo CD is running in):
+
+`argocd app create {{app-name}} --repo {{git-repo-url}} --path {{repo-path}} --dest-server https://kubernetes.default.svc --dest-namespace {{ns}}`
```
Comment (blueskyson) [inline pages/common/argocd-app.md:16]:
```suggestion
`argocd app create {{app_name}} --repo {{git_repo_url}} --path {{path/to/repo}} --dest-server https://kubernetes.default.svc --dest-namespace {{ns}}`
```

File: pages/common/argocd-app.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Delete an application:
+
+`argocd app delete {{app-name}}`
```
Comment (blueskyson) [inline pages/common/argocd-app.md:20]:
```suggestion
`argocd app delete {{app_name}}`
```

File: pages/common/argocd-app.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+- Enable application auto-sync:
+
+`argocd app set {{app-name}} --sync-policy auto --auto-prune --self-heal`
```
Comment (blueskyson) [inline pages/common/argocd-app.md:24]:
```suggestion
`argocd app set {{app_name}} --sync-policy auto --auto-prune --self-heal`
```

File: pages/common/argocd-app.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Preview app synchronization without affecting cluster:
+
+`argocd app sync {{app-name}} --dry-run --prune`
```
Comment (blueskyson) [inline pages/common/argocd-app.md:28]:
```suggestion
`argocd app sync {{app_name}} --dry-run --prune`
```

File: pages/common/argocd-app.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+- Show application deployment history:
+
+`argocd app history {{app-name}} --output {{wide|id}}`
```
Comment (blueskyson) [inline pages/common/argocd-app.md:32]:
```suggestion
`argocd app history {{app_name}} --output {{wide|id}}`
```

File: pages/common/argocd-app.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Rollback application to a previous deployed version by history ID (deleting unexpected resources):
+
+`argocd app rollback {{app-name}} {{history-id}} --prune`
```
Comment (blueskyson) [inline pages/common/argocd-app.md:36]:
```suggestion
`argocd app rollback {{app_name}} {{history_id}} --prune`
```

File: pages/common/argocd.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# argocd
+
+> Command-line interface to control a Argo CD server.
+> Some subcommands such as `argocd app` have their own usage documentation.
+> More information: <https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd/>.
+
+- Login to Argo CD server:
+
+`argocd login --insecure --username {{user}} --password {{password}} {{argocd-server:port}}`
```
Comment (blueskyson) [inline pages/common/argocd.md:9]:
```suggestion
`argocd login --insecure --username {{user}} --password {{password}} {{argocd_server:port}}`
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**